display search result from page 0 even we are page 350

frederic34 opened the pull request.

frederic34 committed:
display search result from page 0 even we are page 350

frederic34 commented:
⚠️ it breaks mass actions...

eldy commented:
Solution may be to add somewhere a test to reset page nb if nb of answer are set to a higher position than number of record in the search result like this ?
if (($page * $limit) > $nbtotalofrecord) { $page = 0 }

fappels committed:
Fix paging when filtering

If total resultset is smaller then paging size (filtering), goto and
load page 0.
If total resultset is smaller the limit, no need to do paging.

fappels committed:
Fix comment

frederic34 committed:
Merge pull request #1 from fappels/frederic34-patch-6

 Fix paging when filtering

eldy committed:
Update myobject_list.php
